src/seedu/addressbook/data/person/AddressPart.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public class AddressPart {
```

Comment:
Missing header comment. All non trivial classes/methods should have javadoc format header comments.